classes/Task.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+// Класс, представляющий сущность "Задание"
+class Task
+{
+    private int $customer_id;
```

Comment:
Писать лучше сразу "правильно". В PHP нотация camel_case используется очень редко в исключительных случаях. По PSR стандартам лучше писать в нотации studlyCaps.
```suggestion
    private int $customerId;
```